components/util/consulting/benefits.tsx

```diff
@@ -9,12 +9,14 @@ const BenefitCard = ({ data, aosType }) => {
   return (
     <article className="px-14 py-11" data-aos={aosType}>
       <figure className="relative float-left h-40 w-40 select-none">
-        <Image
-          src={`/images/benefits/${path.basename(image)}`}
-          sizes="100vw"
-          fill
-          alt="benefit icon"
-        />
+        {image && (
+          <Image
+            src={`/images/benefits/${path.basename(image)}`}
```

Comment:
Can we avoid hard coding the path and just use the one provided?

Reply:
Yes, it is possible to do that. Would you like me to create a variable to store the complete path of the image and then pass it as a prop to the Image component?

Reply:
```suggestion
            src={image}
```
I would've expected this to work

Reply:
Fixed

Reply:
Sorry but you change still has a hardcoded path, it has only been moved around. Did you try the suggestion I posted?

Reply:
It looks like I already made this change on master (there are branch conflicts)

Reply:
I've merged changes from `main` as the method works and avoids hardcoded paths

Reply:
I read it after pushing my change

> It looks like I already made this change on `main` (there are branch conflicts)